src/linker/Linker/DgmlDependencyRecorder.cs

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+//
+// Tracer.cs
```

Comment:
Do you mean remove the copyright portion or just the tracer.cs? This was copied directly from the XML recorder file, I can remove it there too if that's what you mean

Reply:
Answering this, existing files with a different copyright notice keep both notices, in this case since this is a new file only needs the Copyright from .NET foundation.